sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -211,15 +211,15 @@ To make use of an Active Directory principal, one of the available credentials f
 To make use of an Active Directory principal with Azure Storage blob containers, the fully qualified URL to the container must be provided when creating the storage client.  Details about the valid URI formats for accessing Blob storage may be found in [Naming and Referencing Containers, Blobs, and Metadata](https://docs.microsoft.com/rest/api/storageservices/Naming-and-Referencing-Containers--Blobs--and-Metadata#resource-uri-syntax).  
 
 ```C# Snippet:EventHubs_Processor_ReadMe_CreateWithIdentity
-TokenCredential credential = new DefaultAzureCredential();
-
-string blobStorageUrl ="<< FULLY-QUALIFIED CONTAINER URL (like https://myaccount.blob.core.windows.net/mycontainer) >>";
-BlobContainerClient storageClient = new BlobContainerClient(new Uri(blobStorageUrl), credential);
+var credential = new DefaultAzureCredential();
+var blobStorageUrl ="<< FULLY-QUALIFIED CONTAINER URL (like https://myaccount.blob.core.windows.net/mycontainer) >>";
 
 var fullyQualifiedNamespace = "<< FULLY-QUALIFIED EVENT HUBS NAMESPACE (like something.servicebus.windows.net) >>";
 var eventHubName = "<< NAME OF THE EVENT HUB >>";
 var consumerGroup = "<< NAME OF THE EVENT HUB CONSUMER GROUP >>";
 
+BlobContainerClient storageClient = new BlobContainerClient(new Uri(blobStorageUrl), credential);
```

Comment:
Curious why the switch to explicitly call out the type rather than `var`?

Reply:
I honestly have no idea.   Updated that line and the following since the types are clear for both.